_includes/v2.2/app/insecure/sqlalchemy-basic-sample.py

```diff
@@ -1,35 +1,108 @@
-from __future__ import print_function
+import random
+from math import floor
 from sqlalchemy import create_engine, Column, Integer
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
+from cockroachdb.sqlalchemy import run_transaction
 
 Base = declarative_base()
 
+
 # The Account class corresponds to the "accounts" database table.
 class Account(Base):
     __tablename__ = 'accounts'
     id = Column(Integer, primary_key=True)
     balance = Column(Integer)
 
-# Create an engine to communicate with the database. The "cockroachdb://" prefix
-# for the engine URL indicates that we are connecting to CockroachDB.
-engine = create_engine('cockroachdb://maxroach@localhost:26257/bank',
-                       connect_args = {
-                           'sslmode' : 'disable'
-                       })
-Session = sessionmaker(bind=engine)
+
+# Create an engine to communicate with the database. The
+# "cockroachdb://" prefix for the engine URL indicates that we are
+# connecting to CockroachDB using the 'cockroachdb-python' dialect.
```

Comment:
I think I'd just call this `cockroachdb`, not `cockroachdb-python`. The repo is named `cockroachdb-python`, but users won't generally see the repo. They'll use `pip` or `requirements.txt` to install the package, and in that context it's just called `cockroachdb`.

Reply:
Fixed.